src/mmw/js/src/data_catalog/controllers.js

```diff
@@ -14,10 +14,32 @@ var DataCatalogController = {
             'was_data_catalog_visible': true,
         });
 
-        var searchModel = new models.DataCatalogModel();
-        var resources = new models.ResourceCollection();
+        var form = new models.SearchForm();
 
-        var view = new views.DataCatalogWindow(searchModel, resources);
+        var catalogs = new models.Catalogs([
+            new models.Catalog({
```

Comment:
That's an interesting feature. I think I'd prefer to leave this as-is since I can't think of a compelling reason to not specify the type.